### ISSUE TITLE
Test set triplet sample

### ISSUE THREAD

**Issue by hhh920406** [maciejkula/triplet_recommendations_keras#7]
Hi, thank you for your sharing ! you sample the triplet from test set as you do that from training set. But in the real world, we can't know which items are positive or negative. How can we sample the triplet from the test set?

**Comment by maciejkula** [comment]
All the items that the user could have interacted with (bought/rated) are implicitly treated as negatives.
